internal/metrics/metrics.go

```diff
@@ -51,7 +51,7 @@ var BackgroundJobDuration = prometheus.NewHistogramVec(
 		Name:        "provisioning_background_job_duration_ms",
 		Help:        "task queue job duration (ms) by type",
 		ConstLabels: prometheus.Labels{"service": version.PrometheusLabelName, "component": "worker"},
-		Buckets:     []float64{50, 100, 250, 500, 1000, 2500, 6000},
+		Buckets:     []float64{1000, 3000, 7000, 20000, 60000, 60000 * 5, 60000 * 10, 60000 * 20, 60000 * 40},
```

Comment:
Sounds reasonable, will merge, but FYI @adiabramovitch in case you'd not agree lets discuss post fact :)

Reply:
Maybe this is a chance to change this to seconds before we fully commit to SLI/SLOs? Easy change I could amend.